src/scripting/lua_mathx.cpp

```diff
@@ -22,6 +22,9 @@
 #include "lua/lua.h"
 #include "lua/lualib.h"
 
+#include <stdio.h>
+#include <math.h>
```

Comment:
These should be `<cstdio>` and `<cmath>`.

Reply:
It's fixed